Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -171,10 +175,12 @@ def cleanNodeName(self, value):
         cleanValue = cleanValue.replace("\\", "-")
         return cleanValue
 
-    def examineFiles(self, files):
+    def examineFiles(self, files, eps = 1.e-6):
```

Comment:
```suggestion
    def examineFiles(self, files, eps=1.e-6):
```

While we are not yet using "ruff format" (or black), this will be the recommended style.

Reply:
ok thanks!